reference/hoon-expressions/rune/lus.md

```diff
@@ -135,42 +135,74 @@ Any Hoon expression, `q`, may be used to define the arm computation.
 
 ### +* "lustar"
 
-
-Produce a type constructor arm.
+Used to assign an alias to doors and define virtual arms.
 
 ##### Syntax
 
-Regular: **2-fixed**.
+Regular: **variadic**.
 
 ```hoon
-+*  p=term  [q=term]  r=spec
++*  a=term  b=hoon
+    c=term  d=hoon
+    ...
+    e=term  f=hoon
 ```
 
-`p` is the arm name, `q` is the name of the argument for the constructor function enclosed in square brackets, and `r` is a structure expression that defines the constructor function.
+`a`, `c`, `e` are arm names and `b`, `d`, `f` are any Hoon expression. Note that unlike all other
+runes with a variable number of arguments, the list of arguments of `+*` does
+not end with a terminating rune.
+
+`+*` arms must always come at the beginning of the battery, before any other
+type of lus arm.
 
 ##### Discussion
 
-Arms produced by `+*` are essentially type constructors.  They are used to construct new types from ones you already have.  Consider `list`, which takes some type -- e.g., `@`, `tape`, etc. -- and returns a new type, e.g., `(list @)`, `(list tape)`, etc.
+The primary use of `+*` is to assign aliases to doors (see Examples below).
+However, they may also be used to define "virtual arms" which behave just like
+ordinary arms except they allow nesting within cores that requires a specific
```

Comment:
> behave just like ordinary arms

Wondering if this is 100% true.

Reply:
I tried playing around with them a bit and get the impression that I am wrong, but I don't understand what's going on well enough to say what exactly is different.

Reply:
I think deferring to how `+*` turns into `=*`s properly explains what is actually meant by virtual arm. https://github.com/urbit/docs/pull/861/commits/ee469e5acb24eb098227a8fbf68d33d390fd45a4